aiodns/__init__.py

```diff
@@ -6,7 +6,14 @@
     import trollius as asyncio
 import pycares
 
-from . import error
+from typing import (
+    Any,
+    List,
+    Optional,
+)
+
+# TODO: Work out mypy no attribute error
+from . import error # type: ignore
```

Comment:
Seems to be a mypy bug with this kind of import
- The import itself was already in the code